### PR TITLE
Adds blog post, puts blog at /blog and docs at /docs

### PR DESCRIPTION
**Pull request by dwwoelfel**
View the changes on staging: https://oph-dashboard-staging.firebaseapp.com/

Lots of great stuff in here:

- Adds the first draft launch HN post as its own blog post
- The docs live at onegraph.com/docs
- The blog used to be duplicated in `docs.onegraph.com/blog` and `onegraph.com/blog`. Now it lives solely in the docusaurus website and is only available at `onegraph.com/blog`.
- Deploys to staging on every push to non-master. 

- Fixed the green header color when the site gets small

Lots of the changes here are from checking in the generated docs and blog files. This helps us make sure we're not surprised by the code that's generated (would be nice if github had a way of hiding those files, though).

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -98,18 +65,43 @@ jobs:
...
-            - build_docu
           filters:
             branches:
               only: master
+      - staging_deploy:
+          requires:
+            - build
+          filters:
+            branches:
+              ignore: master
```
Comment (sgrove) [inline .circleci/config.yml]: Wondered how to do that.

File: README.md [hunk 2]
```diff
@@ -5,10 +5,28 @@
...
+then navigate to http://localhost:3000/docs or
+http://localhost:3000/website
+
+Make sure to run yarn build-website and check in the generated files
+before pushing, or else circle will reject your changes.
+
 ## Reason + BuckleScript + CRA
 
-BS outputs files that eslint has some problems with. To avoid this, we run both `prettier` and a custom command to prepend an eslint pragma to disable eslint for generated files, since CRA doesn't allow you to customize eslint otherwise. The startup time for prettier is fairly egregious though, adding ~150-300ms to the end of a file compilation 
+BS outputs files that eslint has some problems with. To avoid this, we
```
Comment (sgrove) [inline README.md]: `M-q` in emacs, or did you do this manually? :D

File: public/blog/atom.xml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<feed xmlns="http://www.w3.org/2005/Atom">
+    <id>https://www.onegraph.com/blog</id>
+    <title> Blog</title>
+    <updated>2018-07-18T06:00:00Z</updated>
+    <generator>Feed for Node.js</generator>
```
Comment (sgrove) [inline public/blog/atom.xml]: This is a bit weird, but I suppose it doesn't matter too much.

File: README.md (hunk 2 above)
Comment (dwwoelfel) [reply README.md]: M-q :). Was hoping prettier would do it for me, but no dice :(

**Pull request merged**